File: Assignment 2/if else/Greaterthan10.cpp
#include<iostream>
using namespace std;
class Great_Number
{
    int number;
    public:void greatnum()
    {
        cout<<"Enter a number:\n";
        cin>>number;
        if(number>10)
        {
            cout<<"The number is greater than 10";
        }
        else
        {
            cout<<"The number is less than 10";
        }

    }
};
int main()
{
    Great_Number obj;
    obj.greatnum();
}

File: Java assignment(1-2)/java Assignment 2/Greaterthan10.java
import java.util.Scanner;
class Great_10
{
int num1;
void getData()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter a positive integer: ");
num1=sc.nextInt();
if(num1>10)
{
System.out.println("The number is greater than 10");
}
else
{
System.out.println("The number is less than 10");
}
}
public static void main(String...args)
{
Great_10 obj=new Great_10();
obj.getData();
}
}